tests/bluetooth/tester/src/btp_bap.c

```diff
@@ -524,8 +526,25 @@ static void stream_qos_set(struct bt_bap_stream *stream)
 static void stream_enabled(struct bt_bap_stream *stream)
 {
 	struct audio_stream *a_stream = CONTAINER_OF(stream, struct audio_stream, stream);
+	struct bt_bap_ep_info info;
+	struct bt_conn_info conn_info;
+	int err;
 
 	LOG_DBG("Enabled stream %p", stream);
+
+	err = bt_bap_ep_get_info(stream->ep, &info);
+	if (err) {
+		LOG_ERR("Could not get EP info for stream %p", stream);
```

Comment:
just return here,  if we cannot get ep info we won't be able to properly fill in event data anyway (as info will be left uninitialized).

or maybe send complete event but with failed status?

(same in stream_strated)